ProcessLib/LiquidFlow/Tests.cmake

```diff
@@ -111,6 +110,35 @@ AddTest(
     mesh2D.vtu gravity_driven_adaptive_dt_pcs_1_ts_10_t_300.000000.vtu v_y_ref v_y
 )
 
+AddTest(
+    NAME Adaptive_dt_ThermalConvectionFlow2D
+    PATH StaggeredCoupledProcesses/TH/ThermalConvection2D
+    EXECUTABLE ogs
+    EXECUTABLE_ARGS quad_5500x5500_adaptive_dt.prj
+    WRAPPER time
+    TESTER vtkdiff
+    REQUIREMENTS NOT OGS_USE_MPI
```

Comment:
Added as
```
REQUIREMENTS NOT (OGS_USE_LIS OR OGS_USE_MPI)
```
by amending the commit of ``[dt] Added two more tests about about staggered scheme with adaptive ... ``.